notebooks/xeus-python.svg

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+
+<svg
```

Comment:
GitHub cannot render SVGs in notebooks unfortunately. We need to upload a png for the notebook.